README.md

```diff
@@ -25,3 +27,13 @@ Current examples:
 **6.** [Pager & Syntax Highlighting](https://github.com/jeaye/ncurses-rs/blob/master/examples/ex_6.rs)  
 **7.** [Basic Input & Attributes (Unicode)](https://github.com/jeaye/ncurses-rs/blob/master/examples/ex_7.rs)  
 **8.** [Special ACS Characters](https://github.com/jeaye/ncurses-rs/blob/master/examples/ex_8.rs)  
+
+## Environment variables
+
+Some environment variables are read by `build.rs`:
+
+If set, `NCURSES_RS_RUSTC_LINK_LIB` will be used for `NCURSES_RS_RUSTC_LINK_LIB`.
```

Comment:
erm, unclear for what? for `cargo:rustc-link-lib=` ? or for the lib to link with? 
https://github.com/jeaye/ncurses-rs/blob/3aa22bc279e4929e3ab69d49f75a18eda3e431e9/build.rs#L48-L53